### PR TITLE
Add disk activation to SLEM 5.1 DVD scenario 

### PR DESCRIPTION
**Pull request by mloviska**

### CODE REVIEW

**Review by jlausuch — APPROVED** [review]
LGTM. Let's prove it with a VR

**Comment by mloviska** [comment]
> LGTM. Let's prove it with a VR

Season ticket on a one way ride :man_singer: 
https://openqa.suse.de/tests/7138622#

File: schedule/sle-micro/dvd_on_zvm.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name:           sle_micro_dvd
```
Comment (jlausuch) [inline schedule/sle-micro/dvd_on_zvm.yaml:1]:
```suggestion
name:           sle_micro_dvd_zvm
```

File: schedule/sle-micro/dvd_on_zvm.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+description:    >
+    Maintainer: jalausuch@suse.com, qa-c@suse.de.
```
Comment (jlausuch) [inline schedule/sle-micro/dvd_on_zvm.yaml:3]:
```suggestion
    Maintainer: qa-c@suse.de.
```

File: schedule/sle-micro/dvd_on_zvm.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    SUSE Linux Enterprise Micro tests
+conditional_schedule:
+  maintenance:
+    FLAVOR:
+      'DVD-Updates':
+        - transactional/install_updates
+  boot:
+    ARCH:
```
Comment (jlausuch) [inline schedule/sle-micro/dvd_on_zvm.yaml:11]: since this yaml is only for zVM, can we remove useless variables?

**Comment by mloviska** [comment]
Newest VR: https://openqa.suse.de/tests/7141302#
JG update: https://gitlab.suse.de/qac/qac-openqa-yaml/-/merge_requests/428

**Comment by jlausuch** [comment]
Maintenance jobs will also fail, as it needs reboot I think. But anyway, let's keep it there and deal with that after 5.1 :)
Good job!

**Review by jlausuch — APPROVED** [review]

**Pull request merged**